syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -721,8 +764,8 @@ def unbind(tensor_shares, **kwargs):
 
         module.unbind = unbind
 
-        @overloaded.function
-        def stack(tensors_shares, **kwargs):
+        def cat_and_stack(tensors_shares, cmd, **kwargs):
```

Comment:
nit: maybe the part where the if is needed could be performed in `cat` and `stack` directly?

Reply:
Yes, done.